ote/src/cljs/ote/views/transit_visualization.cljs

```diff
@@ -806,7 +810,15 @@
         ;; Route listing with number of changes
         "Taulukossa on listattu valitussa palvelussa havaittuja muutoksia. Voit valita listalta yhden reitin kerrallaan tarkasteluun. Valitun reitin reitti- ja aikataulutiedot näytetään taulukon alapuolella kalenterissa, kartalla, vuorolistalla ja pysäkkiaikataululistalla."
 
-        [route-changes e! (:gtfs/route-changes changes) selected-route route-hash-id-type]]]
+        [:div.row
+         [:div.col-md-6 ""]
```

Comment:
Out of curiosity, is there a reason not to use `flex` layout or other dynamic approach? Generally that scales a bit better in some cases than forcing to two columns.

Reply:
Changed to use flex